src/_playground/App.scss

```diff
@@ -84,6 +84,27 @@ $menu-spacing: 20px;
             height: 2rem;
         }
     }
+
+    &__screen-reader-only-focusable {
+        background-color: transparent;
+        color: #fff;
+        position: absolute;
+        width: 1px;
+        height: 1px;
+        margin: -1px;
+        padding: 0;
+        overflow: hidden;
+        border: 0;
+
+        &:focus {
+        position: static;
+        width: auto;
+        height: auto;
+        margin: 0;
+        overflow: visible;
+        clip: auto;
+        }
```

Comment:
Minor linting nit: these properties (inside `&:focus`) should be indented one more level.

Reply:
good opportunity to init stylelint here... (but in a separate PR)